### DISCUSSION TITLE
Status Alarms Deleted

### DISCUSSION THREAD

**Discussion by davidkassa** [forwardemail/forwardemail.net#226] [General]
I noticed that there were some SMTP alarms that went off yesterday but the issues have been deleted instead of resolved. Why? This doesn't seem very transparent. Issues were 407, 408, 409.

**Comment by titanism** [comment]
Hi @davidkassa - upptime monitor (the project we use for status page) automatically deletes issues, however the historical uptime is still preserved in the yml files.  Will look into configuring this so it doesn't auto-delete, going to leave this open.  Thank you!
